packages/components/ng-ovh-payment-method/src/components/register/index.html

```diff
@@ -67,6 +67,10 @@
                 data-translate="ovh_payment_method_register_set_as_default_choice_information"
             ></span>
         </oui-message>
+        <p
+            data-ng-if="$ctrl.model.selectedPaymentMethodType.type.paymentType === $ctrl.CREDIT_CARD_PAYMENT_METHOD && $ctrl.hasSpecificCrossBorderSentenceForCardPayment"
```

Comment:
Suggestion: Can you move the check to controller for readability ?

Reply:
@sachinrameshn : fixed !